.github/workflows/merge-check.yml

```diff
@@ -43,7 +43,73 @@ jobs:
         uses: ./.github/actions/tools
 
       - name: Validate Flux Manifests
+        working-directory: ./infrastructure/cluster/flux
         run: |
-          chmod +x ./infrastructure/scripts/flux_validate.sh
-          ./infrastructure/scripts/flux_validate.sh
+          #!/usr/bin/env bash
```

Comment:
If the CI is currently working like this, shouldn't you erase the the `actions/tools/entrypoint.sh` file ?